Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -3038,26 +3664,40 @@ def main():
             headers=headers,
             proxy=proxy,
             block_site_ids=block_site_ids,
-            global_block=global_block
+            global_block=global_block,
         )
 
-        if command == 'test-module':
-            return_results(test_module(client, params.get('isFetch'), first_fetch_time))
-        if command == 'fetch-incidents':
-            fetch_incidents(client, fetch_limit, first_fetch_time, fetch_threat_rank, fetch_site_ids)
-
+        if command == "test-module":
+            return_results(test_module(client, params.get("isFetch"), first_fetch_time))
+        if command == "fetch-incidents":
+            fetch_incidents(
+                client, fetch_limit, first_fetch_time, fetch_threat_rank, fetch_site_ids
+            )
+        # for incoming mirroring (S1 - XSOAR)
+        elif command == "get-remote-data":
+            return_results(get_remote_data_command(client, demisto.args()))
+        elif command == "get-modified-remote-data":
+            return_results(get_modified_remote_data_command(client, demisto.args()))
+
+        # for outcoming mirroring (XSOAR - S1)
+        elif command == "update-remote-system":
+            return_results(update_remote_system_command(client, demisto.args()))
+        elif command == "get-mapping-fields":
+            return_results(get_mappping_fields_command())
```

Comment:
add the `get-remote-data`, `get-modified-remote-data`, `update-remote-system`, `get-mapping-fields` to commands dict